astropy/stats/funcs.py

```diff
@@ -28,7 +28,8 @@
 __doctest_requires__ = {'binom_conf_interval': ['scipy.special'],
                         'poisson_conf_interval': ['scipy.special',
                                                   'scipy.optimize',
-                                                  'scipy.integrate']}
+                                                  'scipy.integrate',
+                                                  'scipy.stats']}
```

Comment:
I think this is wrong, right? I think you should add 'scipy.stats' under a new key called 'biweight_midcovariance'.